main_2024.py

```diff
@@ -277,6 +287,22 @@ def main() -> int:
     # Get Pylance to stop complaining
     assert geolocation_worker_properties is not None
 
+    result, cluster_estimation_worker_properties = worker_manager.WorkerProperties.create(
+        count=1,
+        target=cluster_estimation_worker.cluster_estimation_worker,
+        work_arguments=(MIN_ACTIVATION_THRESHOLD, MIN_NEW_POINTS_TO_RUN, RANDOM_STATE),
+        input_queues=[geolocation_to_cluster_estimation_queue],
+        output_queues=[cluster_estimation_to_main_queue],
+        controller=controller,
+        local_logger=main_logger,
+    )
+    if not result:
+        main_logger.error("Failed to create arguments for Video Input", True)
```

Comment:
Sorry I missed this in the first round of review. Change the error to `Failed to create arguments for Cluster Estimation`.